### ISSUE TITLE
Can't remove separator line in bookmarks

### ISSUE THREAD

**Issue by JoseAlbertoLopezLopez** [IgnorantGuru/spacefm#810]
The option for removing separator line in bookmarks appears blocked.

**Comment by ncmprhnsbl** [comment]
it is possible via the menu bar bookmarks dropdown menu, but not on the bookmarks sidebar.
the latter is then reflected in the former.

**Comment by JoseAlbertoLopezLopez** [comment]
> it is possible via the menu bar bookmarks dropdown menu, but not on the bookmarks sidebar. the latter is then reflected in the former.

No, is not the bookmark side panel, but the Sparator lines that you can add via right click in the bookmarks area -> New -> Separator. That path inserts an horizontal line that cannot be removed because its remove option is blocked.

**Comment by ncmprhnsbl** [comment]
i'll rephrase: it is possible via the menu bar bookmarks dropdown menu, but not on the bookmarks area.
the latter is then reflected in the former.
![Peek 2023-02-25 08-42](https://user-images.githubusercontent.com/5000554/221311158-3edb021a-ee63-4db2-9287-a2c7fe523aca.gif)
not saying this isn't a bug, but there is a work around.

**Comment by JoseAlbertoLopezLopez** [comment]
Thanks! that animation is what I needed. The workaround is very subtle.
